docs/glossary.md

```diff
@@ -0,0 +1,21 @@
+# Glossary
+
+| napari term        | description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | synonym/closely related term                                                |
+|--------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------|
+| canvas             | The canvas in the center of napari viewer contains the visual display of the data passed to napari, including images, points, shapes, and other supported data types.                                                                                                                                                                                                                                                                                                                                                                                                                        |                                                                             |
+| channel            | A channel is an image that each pixel only contains its intensity value (i.e. grayscale image). In fluorescence microscopy, each channel corresponds to a range of wavelengths. An RGB image from digital camera consists of 3 channels - red, green, blue (RGB). In napari, an image dimension that is not spatial or temporal can be considered as channels, and is typically represented as individual [image layer](https://napari.org/tutorials/fundamentals/image.html). See also [this wiki page](https://en.wikipedia.org/wiki/Channel_(digital_image)) for specific examples.       | raster band in geoscience                                                   |
```

Comment:
```suggestion
| channel            | A channel is an image where each pixel only contains its intensity value (i.e. grayscale image). In fluorescence microscopy, each channel corresponds to a range of wavelengths. An RGB image from digital camera consists of 3 channels - red, green, blue (RGB). In napari, an image dimension that is not spatial or temporal can be considered as channels, and is typically represented as individual [image layer](https://napari.org/tutorials/fundamentals/image.html). See also [this wiki page](https://en.wikipedia.org/wiki/Channel_(digital_image)) for specific examples.       | raster band in geoscience                                                   |
```

Reply:
maybe "for which" is better grammar, not sure, but i don't think "that" is right

Reply:
"where" sounds good to me